doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1352 @@
+  - [4.4 Linux Kernel](#44-linux-kernel)
+- [5 CLI](#5-cli)
+  - [5.1 Click CLI](#51-click-based-cli)
+    - [5.1.1 Configuration Commands](#511-configuration-commands)
+    - [5.1.2 Show Commands](#512-show-commands)
+  - [5.2 SONiC CLI](52-#sonic-cli)
+    - [5.2.1 Configuration Commands](521-#configuration-commands)
+    - [5.2.2 Show Commands](#522-show-commands)
+- [6 Serviceability and Debug](#6-serviceability-and-debug)
+- [7 Warm reboot Support](#7-warm-reboot-support)
+- [8 Unit Test Cases ](#8-unit-test-cases)
+
+# List of Tables
+
+[Table 1: Abbreviations](#table-1-abbreviations)
+
+# Revision
+| Rev  |    Date    |       Author        | Change Description                                           |
+|:--:|:--------:|:-----------------:|:------------------------------------------------------------:|
+| 0.1  |  |   Rajesh Sankaran   | Initial version                                              |
+| 0.2  |  | Rajesh Pukhraj Jain | Added Warm boot requirement/design details. Added CLI, Functional, Scalability & Warm boot test cases |
+| 0.3  |  |   Rajesh Sankaran   | Incorporated Review comments                                 |
+| 0.4  |  | Tapash Das, Hasan Naqvi | Added L3 VXLAN details |
+| 0.5  |  | Karthikeyan A | ARP and ND suppression |
+| 0.6  |  | Kishore Kunal | Added Fdbsycnd details |
+| 0.7  |  | Rajesh Sankaran | Click and SONiC CLI added |
+| 0.8 | | Hasan Naqvi | Linux kernel section and fdbsyncd testcases added |
+
+# Definition/Abbreviation
+
+### Table 1: Abbreviations
+
+| **Term** | **Meaning**                               |
+| -------- | ----------------------------------------- |
+| BGP      | Border Gateway Protocol                   |
+| BUM      | Broadcast, Unknown unicast, and Multicast |
+| EVPN     | Ethernet Virtual Private Network          |
+| IMET     | Inclusive Multicast Ethernet Tag          |
+| IRB      | Integrated Routing and Bridging           |
+| L3GW     | Layer 3 Gateway                           |
+| NVO      | Network Virtualization Overlay            |
+| VNI      | VXLAN Network Identifier                  |
+| VRF      | Virtual Routing and Forwarding            |
+| VTEP     | VXLAN Tunnel End point                    |
+| VXLAN    | Virtual Extended LAN                      |
+
+# About this Manual
+
+This document provides general information about the EVPN VXLAN feature implementation based on RFC 7432 and 8365 in SONiC. 
+This feature is incremental to the SONiC.201911 release which is referred to as current implementation in this document.
+
+
+# 1 Introduction and Scope
+
+This document describes the Functionality and High level design of the EVPN VXLAN feature.
+
+The following are the benefits of an EVPN approach for VXLAN.
+
+ - Standards based and Interoperable solution.
+ - Auto discovery of remote VTEPs, Auto provisioning of tunnels and VLANs over VXLAN tunnels.
+ - Support for L2 and L3 VPN services.
+ - Allows for dual homing support.
```

Comment:
But dual homing support is "outside the scope of this document" ?

Reply:
That was an introductory para listing the benefits of EVPN in general and not specific to what is being implemented. The subsequent paragraphs mention that this is not in the scope of the implementation.

Reply:
thanks for the clarification